superset/reports/commands/execute.py

```diff
@@ -207,11 +209,10 @@ def _get_screenshot(self) -> bytes:
         return image_data
 
     def _get_csv_data(self) -> bytes:
-        if self._report_schedule.chart:
-            url = self._get_url(csv=True)
-            auth_cookies = machine_auth_provider_factory.instance.get_auth_cookies(
-                self._get_user()
-            )
```

Comment:
`_get_csv_data` is only called for charts, so the if block is not needed. And even if it where called for non-charts, the call to `get_chart_csv_data` would fail, so I think it's safe to remove the block.